From 2f9f68d2fa2d73545c8ece5841a15ab024a2798b Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Thu, 23 Aug 2018 15:35:04 +0100
Subject: [PATCH 201/220] BCM270X_DT: Sync downstream VCHIQ node w. upstream

Retain a vestigial cache_line_size parameter to improve backwards-
compatibility.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/bcm2708-rpi.dtsi | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2708-rpi.dtsi b/arch/arm/boot/dts/bcm2708-rpi.dtsi
index fe2694a..00160e0 100644
--- a/arch/arm/boot/dts/bcm2708-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2708-rpi.dtsi
@@ -35,7 +35,6 @@
 		usb = &usb;
 		leds = &leds;
 		fb = &fb;
-		vchiq = &vchiq;
 		thermal = &thermal;
 		axiperf = &axiperf;
 	};
@@ -67,12 +66,10 @@
 			status = "disabled";
 		};
 
-		vchiq: vchiq {
+		mailbox@7e00b840 {
 			compatible = "brcm,bcm2835-vchiq";
-			reg = <0x7e00b840 0xf>;
+			reg = <0x7e00b840 0x3c>;
 			interrupts = <0 2>;
-			cache-line-size = <32>;
-			firmware = <&firmware>;
 		};
 
 		vcsm: vcsm {
@@ -95,7 +92,7 @@
 	};
 
 	__overrides__ {
-		cache_line_size = <&vchiq>, "cache-line-size:0";
+		cache_line_size;
 
 		uart0 = <&uart0>,"status";
 		uart1 = <&uart1>,"status";
-- 
1.9.1

